test/utils.jl

```diff
@@ -37,4 +37,13 @@
     u, v = householderbasis(n)
     @test u × v ≈ n ./ norm(n)
   end
+
+  @test mayberound(1.1, 1, 0.2) ≈ 1
+  @test mayberound(1.1, 1, 0.10000000000000001) ≈ 1.1
+  @test mayberound(1.1, 1, 0.05) ≈ 1.1
+
+  @test mayberound(1.1, 1:5, 0.2) ≈ 1
+
+  @test all(mayberound([1.1, 3.3, 3.1, 4.3], 1:5, 0.2) .≈ [1, 3.3, 3, 4.3])
```

Comment:
```suggestion
  @test all(mayberound.([1.1, 3.3, 3.1, 4.3], 1:5, 0.2) .≈ [1, 3.3, 3, 4.3])
```

Reply:
that yields an error without the vector function
 `@test all(mayberound.([1.1, 3.3, 3.1, 4.3], Ref(1:5), 0.2) .≈ [1, 3.3, 3, 4.3])`
would work